scripts/install_obsolete_jsoncpp_1_7_4.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR=$(mktemp -d)
+(
+    cd $TEMPDIR
+    wget https://github.com/open-source-parsers/jsoncpp/archive/1.7.4.tar.gz
+    tar xvzf "1.7.4.tar.gz"
+    cd "jsoncpp-1.7.4"
+    mkdir -p build/debug
+    cd build/debug
+    cmake -DCMAKE_BUILD_TYPE=debug -DBUILD_STATIC_LIBS=ON -DBUILD_SHARED_LIBS=OFF -DARCHIVE_INSTALL_DIR=. -G "Unix Makefiles" ../..
```

Comment:
Is it possible only building static libs could cause an issue with our cmake finding it?

Also as a test we need to roll back the cmake fixes to ensure this install script triggers the problem.

Reply:
This is the commit to be rolled back c8167a9acd8b558e87181a460a3c4fc148e1c43f